draft-davidson-pp-architecture.md

```diff
@@ -103,7 +103,7 @@ participating entities.
 
 # Introduction
 
-The Privacy Pass protocol provides a privacy-preserving mechanism for
+The Privacy Pass protocol provides an anonymity-preserving mechanism for
```

Comment:
Somewhere in the document, it should specify that the anonymity guarantee of Privacy Pass assumes that client IP addresses are either 1) not monitored/stored by the server, even though the client has no mechanism to verify this (otherwise the server can trivially build a lookup table of client IP addresses to the tokens they have used), or 2) an underlying anonymity tool such as Tor is used to hide the client IP address from the server.

Reply:
That's a good point, I've added a subsection to the assumptions we make on the Client as part of the ecosystem to discuss this.